servers/lib/server/handlers/invitetoteambycsv.coffee

```diff
@@ -37,16 +37,16 @@ module.exports.handler = (req, rres) ->
       processInvitations fileName, client, (err) ->
         return respond 500, err if err
 
-        respond 200, "That's all folks!"
+        respond 200, "That's all folk"
 
     return req.pipe busboy
 
 processInvitations = (fileName, client, callback) ->
   fs.readFile fileName, 'utf8', (err, content) ->
-    return callback err if err
+    return callback err, null if err
```

Comment:
I forgot to edit that line, I put my changes to invitetoteambycsvAnalyze.coffee. 
I will remove it.